### PR TITLE
- created CartQuantityTableViewCell

### PR DESCRIPTION
**Pull request by HaroldFS**
- adjusted the cart total to have now negatives

### CODE REVIEW

File: swift/ios-shoppe-demo/Views/CartQuantityTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
+//
+//  File.swift
+//  ios-shoppe-demo
+//
+//  Created by Harold Davis Jr. on 4/9/20.
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Views/CartQuantityTableViewCell.swift]: headers !== FullStory

Comment (HaroldFS) [reply swift/ios-shoppe-demo/Views/CartQuantityTableViewCell.swift]: Ok its hurt me enough lol I will make a PR for this to automatically be done 😭

File: swift/ios-shoppe-demo/Views/CartTableViewController.swift [hunk 2]
```diff
@@ -56,19 +59,19 @@ class CartTableViewController: UITableViewController {
 
             cell.textLabel?.font = UIFont(name: "Halvetica-Bold", size: 30)
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Views/CartTableViewController.swift]: While not in this PR, I noticed a typo on the font face `Helvetica`

File: swift/ios-shoppe-demo/Views/CartTableViewCell.swift [hunk 3]
```diff
@@ -12,14 +12,15 @@ import UIKit
...
+    weak var tableViewController: CartTableViewController?
 
     func setupCartOrder() {
         guard let cartOrderTotal = tableViewController?.order.cartOrderTotal() else {
             return
         }
 
-        orderTotalLabel.text = "$\(cartOrderTotal)"
+        orderTotalLabel.adjustsFontSizeToFitWidth = true
+        orderTotalLabel.text = "$\(abs(cartOrderTotal.distance(to: 0)))"
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Views/CartTableViewCell.swift:23]: What's the need for the intermediate calculation?  Is `cartOrderTotal` not valid as-is?

Comment (HaroldFS) [reply swift/ios-shoppe-demo/Views/CartTableViewCell.swift:23]: No it has the tendency to be negative without that :( I need to do some more research on this

**Review by van-fs — APPROVED** [review]
LGTM

**Pull request merged**